pom.xml

```diff
@@ -44,10 +44,12 @@
   <properties>
     <revision>0.13</revision>
     <changelist>-SNAPSHOT</changelist>
-    <jenkins.version>2.89</jenkins.version>
+    <jenkins.version>2.176.1</jenkins.version>
     <java.level>8</java.level>
     <!--  override crappy OSS default -->
     <surefire.rerunFailingTestsCount>0</surefire.rerunFailingTestsCount>
+    <kubernetes-client-api.version>1.0.0</kubernetes-client-api.version>
```

Comment:
what is this version?  This should be a version that matches the underlying version of the library it provides for developer ease :-(

Reply:
https://github.com/jenkinsci/kubernetes-client-api-plugin/pull/2

Reply:
@jtnord this was my response to the same question from the [jenkinsci/kubernetes-plugin](https://github.com/jenkinsci/kubernetes-plugin/pull/591#issuecomment-528884817)

Reply:
> For example, I feel there is potential to extract or create some common kubernetes client functionality in the kubernetes-client-api plugin 

You should never do that in an API plugin,  create another plugin and depend on the api plugin from it.

> In that case we now are in weird position. Do we continue to use the underlying client api version + some additional plugin version? do we diverge and continue incrementing independently? If you do that then you are essentially back to a "1.0.0" situation because the plugin version does not indicate the k8s client dependency version anymore.

the use case should never happen as per above.

Reply:
Should be resolved by https://github.com/jenkinsci/kubernetes-client-api-plugin/pull/4